src/arrayMethodSort.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callBack = compareFunction;
+
+    if (!callBack) {
+      callBack = function (a, b) {
+        return String(a) > String(b) ? 1 : -1;
+      };
```

Comment:
There is no need to reassign the `compareFunction` to `callBack`. You can directly use `compareFunction` and define the default function if `compareFunction` is not provided. This will make the code cleaner.